docs/source/user-guide/general/metrics_guide.rst

```diff
@@ -48,7 +48,29 @@ Multiclass classification per class:
 *   Precision ``'precision_per_class'``
 *   Recall ``'recall_per_class'``
 
-Regression:
+Token Classification (NLP only)
+_______________________________
+
+Classification metrics averaged over the tokens:
+
+*   Accuracy ``'token_accuracy'``
+*   Precision ``'token_precision_macro'``
+*   Recall ``'token_recall_macro'``
+
+Classification metrics per token class:
+
+*   F1 ``'token_f1_per_class'``
+*   Precision ``'token_precision_per_class'``
+*   Recall ``'token_recall_per_class'``
+
+Object Detection (Vision only)
+______________________________
+
+*   Mean average precision ``'average_precision_per_class'``
+*   Mean average recall  ``'average_recall_per_class'``
+
+Regression
```

Comment:
```suggestion
Regression (Tabular only)
```

Reply:
Why limit? (e.g. regression for CV)

Reply:
We don't currently support regression in CV, as far as I know

Reply:
leaving it as is